src/spikeinterface/preprocessing/scale.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from spikeinterface.core import BaseRecording
+from spikeinterface.preprocessing.basepreprocessor import BasePreprocessor
+
+
+class ScaleTouVRecording(BasePreprocessor):
+    """
+    Scale raw traces to microvolts (µV).
+
+    This preprocessor uses the channel-specific gain and offset information
+    stored in the recording extractor to convert the raw traces to µV units.
+
+    Parameters
+    ----------
+    recording : BaseRecording
+        The recording extractor to be scaled. The recording extractor must
+        have gains and offsets otherwise an error will be raised.
+
+    Raises
+    ------
+    AssertionError
+        If the recording extractor does not have scaleable traces.
+    """
+
+    name = "scale_to_uV"
+
+    def __init__(self, recording: BaseRecording):
+        assert recording.has_scaleable_traces(), "Recording must have scaleable traces"
+        from spikeinterface.preprocessing.normalize_scale import ScaleRecordingSegment
```

Comment:
A more general question out of scope for this PR. Often I get confused about imports not at the top of the file and have an impulse to move them to the top, even if this is very wrong. However I am still always interested in why the import is not at the top, I guess the two reasons can be 1) speed up imports 2) avoid circular imports. Do you think it is in general worth adding comments with the reason for placing such imports or is this too much information / clutter?

Reply:
Yes, in order of relevance:
1) Modularization: that is, people do not have that module (think that pynwb is only required for a small section of the library so most people who use the library don't have it) and we want to avoid an improt error.
2) Improve speed ups like with numba, scipy, etc.
3) Avoid circular imports. This case. 

I can add a comment if you think this would be helpful here.

Reply:
Thanks Herberto I think in this case it is probably out of scope for the PR, I was just kind of raising it as a general thought but with no real purpose 😅 I think it's okay to leave for now and can raise it more generally another time.

Reply:
I like discussing. I do this random question in PRs myself. Feel free to ask whatever!

Reply:
I think a comment like this:

> Modularization: that is, people do not have that module (think that pynwb is only required for a small section of the library so most people who use the library don't have it) and we want to avoid an improt error.
Improve speed ups like with numba, scipy, etc.
Avoid circular imports. This case.

is super useful and I know in a month or 6 months something like this will come up and I won't be able to find this very useful comment to link to :) and I'll have to re-explain that exact same thing in a (no doubt) less succinct way.

thanks Heberto!

Reply:
: D
I am happy to hear it was useful for you!